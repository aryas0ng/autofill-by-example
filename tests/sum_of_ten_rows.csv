11,96
15,94
2,82
5,84
3,81
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,