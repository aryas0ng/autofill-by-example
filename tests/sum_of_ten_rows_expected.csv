11,96
15,94
2,82
5,84
3,81
2,83
4,83
43,85
5,43
6,46
9,46
3,38
4,56
2,55
5,53
2,48
6,46
1,40
8,39
6,31
1,25
21,24
3,3